overlay/make-package-set/simplified.nix

```diff
@@ -6,7 +6,6 @@
 }:
 args@{
   rustChannel,
-  rustChannelSha256 ? null,
```

Comment:
It seems that this argument is rendered unnecessary with @oxalica's Rust overlay, as mentioned in commit https://github.com/cargo2nix/cargo2nix/commit/5cd8d79d407894e46735d81ab84896d2e3f5ee5c. If this is the case, why don't we remove all traces of it from the project? If we're already making breaking changes to the public API by switching overlays, forcing users to upgrade and/or regenerate Nix expressions, why keep a legacy attribute around? Is it required anywhere else?

Reply:
I went ahead and removed all instances in the same amended commit

No remaining or foreseen use